Makefile

```diff
@@ -49,6 +49,7 @@ LCA_IMAGE ?= quay.io/openshift-kni/lifecycle-agent-operator:latest
 LCA_GIT_REPO ?= https://github.com/openshift-kni/lifecycle-agent
 LCA_GIT_BRANCH ?= main
 RELEASE_ARCH ?= x86_64
+RECERT_IMAGE ?= quay.io/edge-infrastructure/recert:v0
```

Comment:
Is `v0` still the best default? I noticed `release-4.15` exists so I'm not sure if that's a better tag? It looks like `v0`, `release-4.15` and `latest` all got updated at the same time

Reply:
in SeedGen if you dont set the recertImage field, it defaults to v0 ... so I just used the same one as a default here :shrug:

Reply:
Here
https://github.com/openshift-kni/lifecycle-agent/blob/92eb15c8747347903ba061229f72520781f5ac88/internal/common/consts.go#L41